user-profile-card.js

```diff
@@ -120,23 +122,28 @@ class UserProfileCard extends LocalizeUserProfileCard(RtlMixin(LitElement)) {
 		const openAlert = !this._isOpen ? this.localize('profileCardClosed') : this.localize('profileCardOpened');
 
 		return html`
-		<d2l-dropdown open-on-hover class="d2l-labs-user-profile-card-dropdown">
-			<d2l-profile-image size="${ifDefined(this._getSize())}"
-			role="button"
-			aria-expanded="${this._isOpen}"
-			aria-haspopup="true"
-			aria-label="${this.localize('profileCardOpener', { displayName : this._displayName })}"
-			class="d2l-labs-user-profile-card-opener d2l-dropdown-opener"
-			href=${this._userHref}
-			.token=${this.token}
-			tabindex="0"
-			@keydown="${this._onOpenerKeypress}"
-			@click="${this._onOpenerClicked}"
-			></d2l-profile-image>
+		<d2l-dropdown
+			slotted-opener
+			open-on-hover="${ifDefined(this.noOpenOnHover ? undefined : true)}"
```

Comment:
```suggestion
			?open-on-hover="${!this.noOpenOnHover}"
```
I think this should do the same thing...